pom.xml

```diff
@@ -134,7 +134,7 @@
         <snakeyaml.version>1.25</snakeyaml.version> <!-- 1.25 matches cxf-jackson 3.3.2 -->
         <!-- Next version of swagger requires changes to how path mapping and scanner injection are done. -->
         <swagger.version>1.5.6</swagger.version>
-        <gson.version>2.5</gson.version>
+        <gson.version>2.7</gson.version>
```

Comment:
add comment that microbean helm needs 2.7

Reply:
Added.